wavelength-plugins/profile.js

```diff
@@ -282,32 +287,119 @@ exports.commands = {
+		},
+
+		"": "help",
+		help: function (target, room, user) {
+			this.parse('/help type');
+		},
+	},
+	typehelp: [
+		"/type set [type] - Sets your Favorite Type.",
+		"/type delete - Removes your Favorite Type.",
+	],
+
+	profilecolor: 'pcolor',
+	pcolor: {
+		set: 'add',
+		add: function (target, room, user) {
+			if (!target) return this.parse('/pcolor help');
+			let color = target.trim();
+			if (color.charAt(0) !== '#') return this.errorReply(`The color needs to be a hex starting with "#".`);
+			Db.profilecolor.set(user, color);
+			this.sendReply(`You have set your profile color to "${color}".`);
+		},
+
+		// For staff
+		forceset: 'forceadd',
+		forceadd: function (target, room, user) {
+			if (!this.can('lock')) return false;
+			if (!target || target.length < 3) return this.parse('/pcolor help');
+			target = target.split(',');
+			let targetUser = target[0].trim();
+			let color = target[1].trim();
+			if (color.charAt(0) !== '#') return this.errorReply(`The color needs to be a hex starting with "#".`);
+			Db.profilecolor.set(targetUser, color);
+			if (Users.get(targetUser)) {
+				Users(targetUser).popup(`|html|You have received profile color from ${WL.nameColor(user.name, true)}.<br />Profile Hex Color: ${color}`);
+			}
+			this.sendReply(`You have set profile color of ${targetUser} to ${color}.`);
+		},
+
+		delete: 'remove',
+		remove: function (target, room, user) {
+			if (!this.can('lock')) return false;
+			let userid = (toId(target));
+			if (!target) return this.parse('/pcolor help');
+			if (!Db.profilecolor.has(userid)) return this.errorReply(`${userid} does not have a profile color set.`);
+			Db.profilecolor.remove(userid);
+			if (Users.get(userid)) {
+				Users(userid).popup(`|html|${WL.nameColor(user.name, true)} has removed your profile color.`);
+			}
+			this.sendReply(`You have removed ${target}'s profile color.`);
+		},
+
+		'': 'help',
+		help: function (target, room, user) {
+			if (!user.autoconfirmed) return this.errorReply("You need to be autoconfirmed to use this command.");
+			if (!this.canTalk()) return this.errorReply("You cannot do this while unable to talk.");
+			if (!this.runBroadcast()) return;
+			return this.sendReplyBox(
+				'<center><code>/profilecolor</code> commands<br />' +
+				'All commands are nestled under the namespace <code>pcolor</code>.</center>' +
+				'<hr width="100%">' +
+				'- <code>[set|add] [hex color]</code>: set your profile color.' +
+				'- <code>[forceset|forceadd] [username], [hex color]</code>: Sets a user\'s profile color. Requires: % or higher.' +
+				'- <code>[remove|delete] [username]</code>: Removes a user\'s profile color and erases it from the server. Requires: % or higher.'
+			);
+		},
+	},
+
 	bg: 'background',
 	background: {
 		set: 'setbg',
 		setbackground: 'setbg',
 		setbg: function (target, room, user) {
-			if (!this.can('lock')) return false;
+			if (!this.can('broadcast')) return false;
```

Comment:
leave at %, setup tokens (which ik your doing in another PR)

Reply:
Ah I keep forgetting when I use my personal profile code a lot of these errors are from that